src/lib/index.ts

```diff
@@ -355,6 +379,22 @@ class ServerlessIamPerFunctionPlugin {
         policyStatements.push(s);
       }
     }
+
+    // add iamPermissionsBoundary
+    const customIamRolesPerFunction = this.getCustomIamRolesPerFunction();
+    if (this.hasPermissionsBoundaryDefined(functionObject)){
+
+      functionIamRole.Properties.PermissionsBoundary = {
+        'Fn::Sub': functionObject.iamPermissionsBoundary || customIamRolesPerFunction.iamGlobalPermissionsBoundary,
```

Comment:
Please update function properties validation schema [here](https://github.com/functionalone/serverless-iam-roles-per-function/blob/v3.0.2/src/lib/index.ts#L55)

Reply:
`customIamRolesPerFunction` variable might be `false` in case of requested changes or `undefined` in current state.

Reply:
updated